exchanges/stream/buffer/buffer.go

```diff
@@ -260,6 +245,15 @@ func (w *Orderbook) processObUpdate(o *orderbookHolder, u *orderbook.Update) err
 			return o.ob.Invalidate(err)
 		}
 		o.updateID = u.UpdateID
+	} else if o.ob.VerifyOrderbook {
```

Comment:
Add coverage and I'm happy 🎉 
```
        holder.checksum = nil
	holder.updateIDProgression = false
	hob := holder.ob[key.PairAsset{Base: cp.Base.Item, Quote: cp.Quote.Item, Asset: asset.Spot}]
	hob.ob.VerifyOrderbook = true
	err = holder.Update(&orderbook.Update{
		UpdateTime: time.Now(),
		Asset:      asset.Spot,
		Asks:       []orderbook.Tranche{{Price: 999999}},
		Pair:       cp,
	})
	if err != nil {
		t.Fatal(err)
	}
```

Reply:
Thanks for that; had to add something a little bit extra because I just noticed one method wasn't actually going through the checks at all. [here](https://github.com/thrasher-corp/gocryptotrader/pull/1682/commits/311fc7704e7615521d8cc4fc4b8f7eb74268a571)

Reply:
Nice catch. I was just brute forcing to get coverage